apps/parachain-tracker/src/app/app.component.css

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
What's going on here? 👀

Reply:
oh no

Reply:
I think it is not getting`variables.scss`